Makefile

```diff
@@ -33,6 +51,12 @@ code/run:
 test/unit:
 	@./scripts/ci/unit_test.sh
 
+.PHONY: test/e2e/prow
+test/e2e/prow: setup/prow cluster/prepare
+	@echo Running e2e tests:
+	GO111MODULE=on ./operator-sdk test local ./test/e2e --up-local --namespace $(NAMESPACE) --go-test-flags "-timeout=60m -v"
```

Comment:
Not sure why you are enabling GO111MODULE, more likely you need to disable them since we are using dep

Reply:
Yep. 👍